app.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Portfolio</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <!-- 
+      i used a fontawesome library to import my navigation icon
+     -->
+     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+     <!-- 
+      linked my css file
+      -->
+     <link rel="stylesheet" href="./app.css">
+     <!-- 
+      downloaded inter font from google font this is the link
+      -->
+      <link href="https://fonts.googleapis.com/css2?family=Inter:wght@500;600;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header class="head">
+    <div class="heading_container">
+      <p class="head_heading">Welcome</p>
+    </div>
+    <span class="head_icon_holder">
+      <i class="fa fa-bars icon"></i>
+    </span>
+  </header>    
+  <section class="main">
+    <figure class="figure1">
+      <img src="./assets/main/workingone.png" alt="block1" >
+    </figure>
+    <figure class="figure2">
+      <img src="./assets/main/workingtwo.png" alt="block2">
+    </figure>
+    <figure class="figure3">
+      <img src="./assets/main/workingthree.png" alt="block3">
+    </figure>
+    <figure class="figure4">
+      <img src="./assets/main/workingfour.png" alt="block4">
+    </figure>
+    <figure class="figure5">
+      <img src="./assets/main/workingfive.png" alt="block5">
+    </figure>
+    <article class="main_article">
+        <h2 class="main_article_heading">
+          Hey there. I’m Mariane
+          <u><br> I’m a software developer</u>
+        </h2>
+      <p class="main_article_paragraph"> 
+        I can help you build a product , 
+        feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded, 
+        don’t hestiate to contact me.
+      </p>
+    </article>
+    <nav class="links">
+      <ul>
+        <li><i class="fa fa-twitter"></i></li>
+        <li><i class="fa fa-linkedin"></i></li>
+        <li><i class="fa fa-medium"></i></li>
+        <li><i class="fa fa-github"></i></li>
+        <li><i class="fa fa-angellist"></i></li>
+      </ul>
+    </nav>
```

Comment:
- [ ] There is no need to wrap the social icon list here in a `<nav>` tag as they are not navigation items